routers/api/v1/repo/issue_label.go

```diff
@@ -173,7 +173,7 @@ func DeleteIssueLabel(ctx *context.APIContext) {
 
 	label, err := models.GetLabelInRepoByID(ctx.Repo.Repository.ID, ctx.ParamsInt64(":id"))
```

Comment:
I think org labels should be considered here as well?

Reply:
Hmm yes they should! Will think on it this probably needs some slight refactoring to get the label right just based on id

Reply:
Fixed this one and the more complicated all issues search api endpoint too